lib/content-services/src/lib/content-node-selector/content-node-selector.component.ts

```diff
@@ -26,14 +26,18 @@ import { ContentNodeSelectorComponentData } from './content-node-selector.compon
 import { NodeEntryEvent } from '../document-list/components/node.event';
 import { NodeAction } from '../document-list/models/node-action.enum';
 import { OverlayContainer } from '@angular/cdk/overlay';
+import { Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
 
 @Component({
     selector: 'adf-content-node-selector',
     templateUrl: './content-node-selector.component.html',
     styleUrls: ['./content-node-selector.component.scss'],
     encapsulation: ViewEncapsulation.None
 })
-export class ContentNodeSelectorComponent implements OnInit {
+export class ContentNodeSelectorComponent implements OnInit, OnDestroy {
+    private onDestroy$ = new Subject<boolean>();
```

Comment:
No need to set boolean for Subject as boolean value is skipped. It can be just void as it is enough that subject calls 'next' to unsubscribe